pkgs/by-name/bo/boring/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
```

Comment:
suggestion (non-blocking): Using `finalAttrs` seems a modern style for this case.